### ISSUE TITLE
Update COMRADES documentation for EU: due soon!

### ISSUE THREAD

**Issue by Shadrock** [ushahidi/platform#2004]
### Need
The EU Review has requested that we re-submit our last report ([here on the Ushahidi G-Drive](https://drive.google.com/open?id=0B0fInrbWAUMsSjdUbVc0MFBmVTQ)) with:

> "some UML use case and activity/sequence diagrams for the different users of the COMRADES platform."

At present the only diagram in the report is the brief one that @willdoran made to diagram accessing YODIE via web hooks. I don't expect this to be too complicated but I'm not sure... @jshorland can you please advise: do we need a call or does this make sense as is?

### Due Date
September 5th, 2017

**Comment by Shadrock** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [COMRADES deliverable corrections.](https://trello.com/c/gDwK10nF/222-comrades-deliverable-corrections)

**Comment by willdoran** [comment]
@Shadrock @jshorland I will take a look and see if I can find existing uml/diagrams. @rjmackay do you know of some already?

**Comment by willdoran** [comment]
rjmackay
[17:21] 
I don't think we have proper UML version


[17:21] 
Do they want specific ones for COMRADES and the integrations?


[17:21] 
Sequence diagrams might be specific, but use case diagrams are probably pretty generic.

robbie [17:22] 
I used a mix of https://www.planttext.com/ and  http://nomnoml.com/ to make my other diagrams... although they're both kinda fiddly

will [17:24] 
I’m not totally sure, I had interpreted it as - UML diagrams showing the user flow + the affect of the comrades integrations

@shadrock is that correct? Secondly, are there distinct user groups(election monitors, human rights defenders, etc) that will need to be diagramed? Or is it more ‘How does a user interact with the Platform + the COMRADES integrations?’

So 
- User wants to accomplish X
- They do Y
- System does Z
- They can then do X1
…
